project/Settings.scala

```diff
@@ -92,6 +92,14 @@ object Settings {
 
   lazy val entrypoint = "/opt/docker/bin/entrypoint.sh"
 
+  lazy val entrypointCmd =
+    s"""echo '#!/bin/bash\\numask 002;
+       |until /opt/docker/bin/server; do
+       |  echo 'Welder crashed. Respawning..' >&2
+       |  sleep 1
+       |done' > $entrypoint
```

Comment:
maybe just replace this with `entrypoint`?
```suggestion
lazy val entrypointCmd =
    s"""echo '#!/bin/bash\\numask 002;
       |until /opt/docker/bin/server; do
       |  echo 'Welder crashed. Respawning..' >&2
       |  sleep 1
       |done' > /opt/docker/bin/entrypoint.sh
```